src/api/app/helpers/webui/notification_description_helper.rb

```diff
@@ -0,0 +1,83 @@
+module Webui::NotificationDescriptionHelper
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # TODO: Content of ViewComponent. Move to sub-classes once STI is set.
+  def description(notification)
```

Comment:
Why not move this description into each object, in the same fashion @srbaker did [here](https://github.com/openSUSE/open-build-service/pull/16326/commits/ed112b35c2ab3ba36acc78441ecf49efb476dd20#diff-2c70c5303a618d03dd5d367484732e3ec2305b0354eb890c62bd28085e6251d8R6)

Reply:
This is only the first step towards that, which is already implemented in the draft PR.

See an example [here](https://github.com/openSUSE/open-build-service/pull/16222/files#diff-1ad94b8ff6161b4b82ffec2acd6d48aefb0e59da2dc628557956236286b2af37R1).

Reply:
Awesome! I'll shut up then :sweat_smile: